### PR TITLE
Add organisation page

### PR DESCRIPTION
**Pull request by tinashechiraya**
this is linked to issue:
https://github.com/kartoza/africa_rangeland_watch/issues/50

organisations_on_mobile
organisation_page

### CODE REVIEW

File: django_project/frontend/src/components/SideBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import React from "react";
+import { MenuItem, Menu } from "react-pro-sidebar";
+import { HamburgerIcon } from "@chakra-ui/icons";
+import { useLocation, useNavigate } from "react-router-dom";
+
+interface Props extends ChakraProps {
+  className?: string;
+}
+
+export default function Sidebar1(props: Props) {
```
Comment (dimasciput) [inline django_project/frontend/src/components/SideBar/index.tsx:20]: @tinashechiraya why do you need to a number here?

File: django_project/frontend/package.json [hunk 2]
```diff
@@ -20,6 +20,8 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-helmet": "^6.1.0",
+    "react-icons": "^5.3.0",
```
Comment (dimasciput) [inline django_project/frontend/package.json:23]: can't we use chakra-icons?

File: django_project/frontend/src/pages/OrganisationInformation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                      {orgKey}
+                    </Tab>
+                  ))}
+                </TabList>
+              </Box>
+
+              <Divider mb={6} borderColor="black" borderWidth="1px" width={{base:"auto", md:"99%"}}/>
+
+              <TabPanels>
+                {Object.values(organizations).map((organization, index) => (
```
Comment (dimasciput) [inline django_project/frontend/src/pages/OrganisationInformation/index.tsx:101]: let's use state for organizations, so when we have real data it's going to be easier to implement.

**Comment by tinashechiraya** [comment]
> Sidebar1

resolving these comments

File: django_project/frontend/src/pages/OrganisationInformation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+    invitations: [],
+  },
+  org2: {
+    members: [],
+    invitations: [],
+  },
+};
+
+export default function OrganisationInformation() {
+  const [isSidebarOpen, setSidebarOpen] = useState(true);
```
[the remark this answers is not in the document]
Comment (tinashechiraya) [reply django_project/frontend/src/pages/OrganisationInformation/index.tsx]: @tinashechiraya its not neccessary here , the idea was to keep it open and close it when switch to mobile but this logic is handled differently without requiring this variable ...the code for this is in the sidebar component itself.

**Review by dimasciput — APPROVED** [review]

**Pull request merged**